schemas/endpoints/rebalance.yaml

```diff
@@ -0,0 +1,169 @@
+        schema:
+          type: string
+          default: VALID_WINDOWS
+      - name: verbose
+        in: query
+        description: Return detailed state information
+        schema:
+          type: boolean
+          default: false
+      - name: exclude_recently_demoted_brokers
+        in: query
+        description: Whether to allow leader replicas to be moved to recently demoted broker
+        schema:
+          type: boolean
+          default: false
+      - name: exclude_recently_removed_brokers
+        in: query
+        description: Whether to allow leader replicas to be moved to recently removed broker
+        schema:
+          type: boolean
+          default: false
+      - name: replica_movement_strategies
+        in: query
+        description: Replica movement strategies to use
+        schema:
+          type: array
+          items:
+            type: string
+      - name: ignore_proposal_cache
+        in: query
+        description: Whether to ignore the cached proposal or not
+        schema:
+          type: boolean
+          default: false
+      - name: destination_broker_ids
+        in: query
+        description: Whether to allow leader replicas to be moved to recently demoted broker
+        schema:
+          type: array
+          items:
+            type: string
+      - name: kafka_assigner
+        in: query
+        description: Whether to use Kafka assigner mode to general proposal
+        schema:
+          type: boolean
+          default: false
+      - name: rebalance_disk
+        in: query
+        description: Whether to balance load between brokers or between disks within broker
+        schema:
+          type: boolean
+          default: false
+      - name: review_id
+        in: query
+        description: Review id for 2FA
+        schema:
+          type: string
+      - name: replication_throttle
+        in: query
+        description: Upper bound on the bandwidth used to move replicas
+        schema:
+          type: integer
+          format: int32
+      - name: execution_progress_check_interval_ms
+        in: query
+        description: Execution progress check interval in milliseconds
+        schema:
+          type: integer
+          format: int32
+      - name: reason
+        in: query
+        description: Reason for request
+        schema:
+          type: string
+    responses:
+      '200':
+        description: |-
+          Successful rebalance response
+        content:
+          application/json:
+            schema:
+              $ref: '../responses/optimzationResult.yaml#/OptimizationResult'
+          text/plain:
+            schema:
+              type: string
+      '202':
+        description: |-
+          Rebalancing in progress
+        content:
+          application/json:
+            schema:
+              $ref: '../responses/progressResult.yaml#/ProgressResult'
+          text/plain:
+            schema:
+              type: string
```

Comment:
we are missing[ error response](https://github.com/linkedin/cruise-control/blob/d3619c650aaff1f3750a7d7b0f3ec5ab5f1c4741/cruise-control/src/main/java/com/linkedin/kafka/cruisecontrol/servlet/KafkaCruiseControlServlet.java#L129) here, I think we can add a default response for all error response since they share the same schema.